src/components/side-panel/data-selection/DataSelection.tsx

```diff
@@ -3,6 +3,7 @@ import ChartContext, { ChartContextProps } from "../../../context/ChartContext";
 import DimensionSelection from "./dimension-selection/DimensionSelection";
 import "./data-selection.css";
 import SeriesSelector from "../property-inputs/SeriesSelector";
+import ConfidenceSelector from "../property-inputs/ConfidenceSelector";
```

Comment:
I'd consider `ConfidenceIntervalSelector` and `../property-inputs/ConfidenceIntervalSelector`- data is mad and there will be other means of measuring the confidence/accuracy of it. Verbose and specific is typically your future friend.

Reply:
Great suggestion, I've made the switch over to ConfidenceIntervalSelector